dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/"
+COPYRIGHT="2002-2010 Red Hat Inc.
+	2003 David Zeuthen
+	2002-2003 CodeFactory AB
+	2003 James Willcox
+	2004 Rob Taylor
+	2004 Anders Carlsson
+	2005 Colin Walters
+	2006 Steve Frécinaux
+	2009-2011 Nokia Corporation
+	2010-2012 Mike Gorse
+	2016 Simon McVittie
+	2005-2019 Collabora Ltd."
+LICENSE="MIT
+	AFL v2.1
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://dbus.freedesktop.org/releases/dbus-python/dbus-python-$portVersion.tar.gz"
+CHECKSUM_SHA256="b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
+SOURCE_DIR="dbus-python-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dbuspython$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libdbus_1$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:bash
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+PYTHON_PACKAGES=(python3 python36)
+PYTHON_VERSIONS=(3.6 3.7)
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku\n\
+	${portName} == $portVersion base\n\
+	lib:libdbus_1$secondaryArchSuffix\n\
+	lib:libglib_2.0$secondaryArchSuffix\n\
+	cmd:python$pythonVersion\
+	\""
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:python$pythonVersion"
+done
+
+BUILD()
+{
+	autoreconf -vfi
+
+	for i in "${!PYTHON_PACKAGES[@]}"; do
+		pythonPackage=${PYTHON_PACKAGES[i]}
+		pythonVersion=${PYTHON_VERSIONS[$i]}
+		python=python$pythonVersion
+
+		cd "$sourceDir"
+		rm -rf build-$pythonPackage
+		mkdir build-$pythonPackage
```

Comment:
These can just be

```
mkdir -p build-$pythonPackage
```

There aren't much harm in keeping those folders, and if we want to remove them one can do `haikuporter -c` (which is the recommended way).